src/main/resources/db/migration/oracle/V1.0.6.0.1__schema-add-analysis_execution_password.sql

```
@@ -1,44 +1,42 @@
-CREATE TABLE IF NOT EXISTS ${ohdsiSchema}.cca_execution_ext (
+CREATE TABLE ${ohdsiSchema}.cca_execution_ext (
   cca_execution_id INTEGER NOT NULL,
-  update_password  VARCHAR
+  update_password  CHARACTER VARYING(255)
 );
 
-ALTER TABLE IF EXISTS ${ohdsiSchema}.input_files
-  DROP CONSTRAINT IF EXISTS fk_sof_cca_execution;
+ALTER TABLE ${ohdsiSchema}.input_files DROP CONSTRAINT fk_sof_cca_execution;
 
-ALTER TABLE IF EXISTS ${ohdsiSchema}.output_files
-  DROP CONSTRAINT IF EXISTS fk_sif_cca_execution;
+ALTER TABLE ${ohdsiSchema}.input_files DROP CONSTRAINT fk_sif_cca_execution;
```

This table does not exist yet.

I believe this was because the 'alter table if exists' were replaced with straight alters, but we should assume that no tables exist yet (since they'd only be created by the migration process), and so we should not have any need for 'if exists' statements anywhere. If ```input_files``` already exists, then the migration should throw an error if it tries to create it.  This will catch environmental errors.